assets/i18n/en.json

```diff
@@ -163,11 +163,17 @@
     "I18N_CLASSROOM_CALLOUT_HEADING_2": "Introducing: The Oppia Classroom",
     "I18N_CLASSROOM_CALLOUT_PARAGRAPH_1": "Check out the first comprehensive course in the all-new Oppia Classroom! Curated lessons - reviewed by educators - so that you can master basic math skills in topics ranging from Place Values to Multiplication and Division.",
     "I18N_CLASSROOM_MATH_TITLE": "Math",
+	"I18N_CLASSROOM_PAGE_ALREADY_KNOW_SOME_MATH_HEADING": "Already know some math?",
```

Comment:
Something seems off about the indentation here -- please check the "Files Changed" tab of your PR. Ditto for all the other newly-added lines.

Reply:
![image](https://user-images.githubusercontent.com/57531197/201913314-2970a8da-e767-4206-b1c2-95e45e0558b2.png)

Yes, this seems a bit off. But in the local changes, the indentation is correct.

Reply:
Updated